src/Host/Monitor/Impl/MainWindow.xaml

```diff
@@ -1,21 +1,21 @@
 ﻿<Window x:Class="Microsoft.R.Host.Monitor.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:Microsoft.R.Host.Monitor"
-        mc:Ignorable="d"
-        Title="R Host Monitor" Height="350" Width="525">
-    <Grid>
+        Title="R Remote Services Monitor" Height="350" Width="560" ResizeMode="NoResize"
+        Background="#FFFCFCFC">
```

Comment:
Well, it's up to Windows to fix their theme :)

We could style the status field though. So long as it's both background and text color, so it's always legible.